dependencies.mk

```diff
@@ -29,26 +28,17 @@ dependencies: $(TRANSIFEX_PO_FILES)
 $(HOME)/.transifexrc:
 	mkdir --parent $(dir $@)
 	echo "[https://www.transifex.com]" > $@
+	echo "api_hostname  = https://api.transifex.com" >> $@
+	echo "rest_hostname = https://rest.api.transifex.com" >> $@
 	echo "hostname = https://www.transifex.com" >> $@
-	echo "username = c2c" >> $@
-	echo "password = c2cc2c" >> $@
-	echo "token =" >> $@
+	echo "token = 1/dc02578696187cc29e4e6486f8611fdbfe60b235" >> $@
```

Comment:
Should we commit this token in a public repository.

Reply:
This user didn't has any rights, it's just because Transifex just works with connected user ...